doc/getting-started/building-from-source.rst

```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
 
 The build results are found in the ``build`` directory. You can specify
 a different Output folder by setting the ``O`` ``make`` parameter,
```

Comment:
```suggestion
a different build folder by setting the ``O`` ``make`` parameter,
```
We should call this a `build` folder instead IMHO, @dbkinder , thoughts?

Reply:
Done!